Bottom: 8a8233d3202fa324442e33a10ac972898abb84ad
Top:    e7686e967fd92cdf5a4e5cc33bfb257e0b801b51
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2013-01-03 19:26:54 +0000

Temporarily undo a few find.c changes


---

diff --git a/src/find.c b/src/find.c
index 6fba1da..70cc17f 100644
--- a/src/find.c
+++ b/src/find.c
@@ -257,7 +257,7 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-//static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
+static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
 static bool User = false;    /* user action causing this */
 static bool drc = false;     /* whether to stop if finding something not found */
@@ -3461,7 +3461,7 @@ drc_callback (DataType *data, LayerType *layer, PolygonType *polygon,
   PinType *pin = (PinType *) ptr2;
   PadType *pad = (PadType *) ptr2;
 
-  SetFubarThing (type, ptr1, ptr2, ptr2);
+  SetThing (type, ptr1, ptr2, ptr2);
 
   switch (type)
     {
@@ -3646,7 +3646,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, line);
             DrawLine (layer, line);
             drcerr_count++;
-            SetFubarThing (LINE_TYPE, layer, line, line);
+            SetThing (LINE_TYPE, layer, line, line);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Line width is too thin"),
@@ -3690,7 +3690,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, arc);
             DrawArc (layer, arc);
             drcerr_count++;
-            SetFubarThing (ARC_TYPE, layer, arc, arc);
+            SetThing (ARC_TYPE, layer, arc, arc);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Arc width is too thin"),
@@ -3735,7 +3735,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, pin);
             DrawPin (pin);
             drcerr_count++;
-            SetFubarThing (PIN_TYPE, element, pin, pin);
+            SetThing (PIN_TYPE, element, pin, pin);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pin annular ring too small"),
@@ -3767,7 +3767,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, pin);
             DrawPin (pin);
             drcerr_count++;
-            SetFubarThing (PIN_TYPE, element, pin, pin);
+            SetThing (PIN_TYPE, element, pin, pin);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pin drill size is too small"),
@@ -3810,7 +3810,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, pad);
             DrawPad (pad);
             drcerr_count++;
-            SetFubarThing (PAD_TYPE, element, pad, pad);
+            SetThing (PAD_TYPE, element, pad, pad);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Pad is too thin"),
@@ -3855,7 +3855,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, via);
             DrawVia (via);
             drcerr_count++;
-            SetFubarThing (VIA_TYPE, via, via, via);
+            SetThing (VIA_TYPE, via, via, via);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Via annular ring too small"),
@@ -3887,7 +3887,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, via);
             DrawVia (via);
             drcerr_count++;
-            SetFubarThing (VIA_TYPE, via, via, via);
+            SetThing (VIA_TYPE, via, via, via);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Via drill size is too small"),
@@ -3930,7 +3930,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, line);
             DrawLine (layer, line);
             drcerr_count++;
-            SetFubarThing (LINE_TYPE, layer, line, line);
+            SetThing (LINE_TYPE, layer, line, line);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
             violation = pcb_drc_violation_new (_("Silk line is too thin"),
@@ -3981,7 +3981,7 @@ DRCAll (void)
             SET_FLAG (SELECTEDFLAG, element);
             DrawElement (element);
             drcerr_count++;
-            SetFubarThing (ELEMENT_TYPE, element, element, element);
+            SetThing (ELEMENT_TYPE, element, element, element);
             LocateError (&x, &y);
             BuildObjectList (&object_count, &object_id_list, &object_type_list);
